--- nfs-utils-1.3.4/utils/nfsidmap/nfsidmap.c.0103~	2016-12-07 00:55:07.497215538 +0100
+++ nfs-utils-1.3.4/utils/nfsidmap/nfsidmap.c	2016-12-07 00:56:17.520410732 +0100
@@ -431,7 +431,7 @@ int main(int argc, char **argv)
 
 	xlog_stderr(verbose);
 	if ((argc - optind) != 2) {
-		xlog_warn("Bad arg count. Check /etc/request-key.conf");
+		xlog_warn("Bad arg count. Check /etc/request-key.d/id_resolver.conf");
 		xlog_warn(usage, progname);
 		return EXIT_FAILURE;
 	}
