--- nfs-utils-2.2.1/utils/statd/sm-notify.man.0100~	2017-11-02 15:53:53.000000000 +0100
+++ nfs-utils-2.2.1/utils/statd/sm-notify.man	2017-11-05 12:33:30.558276592 +0100
@@ -184,7 +184,7 @@ where NSM state information resides.
 If this option is not specified,
 .B sm-notify
 uses
-.I /var/lib/nfs
+.I /var/lib/nfs/statd
 by default.
 .IP
 After starting,
@@ -330,13 +330,13 @@ Currently, the
 command supports sending notification only via datagram transport protocols.
 .SH FILES
 .TP 2.5i
-.I /var/lib/nfs/sm
+.I /var/lib/nfs/statd/sm
 directory containing monitor list
 .TP 2.5i
-.I /var/lib/nfs/sm.bak
+.I /var/lib/nfs/statd/sm.bak
 directory containing notify list
 .TP 2.5i
-.I /var/lib/nfs/state
+.I /var/lib/nfs/statd/state
 NSM state number for this host
 .TP 2.5i
 .I /proc/sys/fs/nfs/nsm_local_state
--- nfs-utils-2.2.1/utils/statd/statd.man.0100~	2017-11-02 15:53:53.000000000 +0100
+++ nfs-utils-2.2.1/utils/statd/statd.man	2017-11-05 12:34:12.365152799 +0100
@@ -253,7 +253,7 @@ where NSM state information resides.
 If this option is not specified,
 .B rpc.statd
 uses
-.I /var/lib/nfs
+.I /var/lib/nfs/statd
 by default.
 .IP
 After starting,
@@ -425,13 +425,13 @@ If set to a positive integer, has the sa
 .IR \-\-no\-notify .
 .SH FILES
 .TP 2.5i
-.I /var/lib/nfs/sm
+.I /var/lib/nfs/statd/sm
 directory containing monitor list
 .TP 2.5i
-.I /var/lib/nfs/sm.bak
+.I /var/lib/nfs/statd/sm.bak
 directory containing notify list
 .TP 2.5i
-.I /var/lib/nfs/state
+.I /var/lib/nfs/statd/state
 NSM state number for this host
 .TP 2.5i
 .I /var/run/run.statd.pid
