diff -up nfs-utils-2.6.2/support/export/auth.c.omv~ nfs-utils-2.6.2/support/export/auth.c
--- nfs-utils-2.6.2/support/export/auth.c.omv~	2022-10-28 16:38:34.658522531 +0200
+++ nfs-utils-2.6.2/support/export/auth.c	2022-10-28 16:38:37.467541450 +0200
@@ -82,7 +82,7 @@ check_useipaddr(void)
 }
 
 unsigned int
-auth_reload()
+auth_reload(void)
 {
 	struct stat		stb;
 	static ino_t		last_inode;
diff -up nfs-utils-2.6.2/support/export/v4root.c.omv~ nfs-utils-2.6.2/support/export/v4root.c
--- nfs-utils-2.6.2/support/export/v4root.c.omv~	2022-10-28 16:38:45.011592232 +0200
+++ nfs-utils-2.6.2/support/export/v4root.c	2022-10-28 16:38:47.643609940 +0200
@@ -198,7 +198,7 @@ static int v4root_add_parents(nfs_export
  * looking for components of the v4 mount.
  */
 void
-v4root_set()
+v4root_set(void)
 {
 	nfs_export	*exp;
 	int	i;
diff -up nfs-utils-2.6.2/support/export/xtab.c.omv~ nfs-utils-2.6.2/support/export/xtab.c
--- nfs-utils-2.6.2/support/export/xtab.c.omv~	2022-10-28 16:38:24.938457026 +0200
+++ nfs-utils-2.6.2/support/export/xtab.c	2022-10-28 16:38:28.078478195 +0200
@@ -135,7 +135,7 @@ xtab_write(char *xtab, char *xtabtmp, ch
 }
 
 int
-xtab_export_write()
+xtab_export_write(void)
 {
 	return xtab_write(etab.statefn, etab.tmpfn, etab.lockfn, 1);
 }
diff -up nfs-utils-2.6.2/support/nfsidmap/regex.c.omv~ nfs-utils-2.6.2/support/nfsidmap/regex.c
--- nfs-utils-2.6.2/support/nfsidmap/regex.c.omv~	2022-10-28 16:39:46.330003677 +0200
+++ nfs-utils-2.6.2/support/nfsidmap/regex.c	2022-10-28 16:39:49.512024967 +0200
@@ -542,7 +542,7 @@ struct trans_func regex_trans = {
 	.gss_princ_to_grouplist	= regex_gss_princ_to_grouplist,
 };
 
-struct trans_func *libnfsidmap_plugin_init()
+struct trans_func *libnfsidmap_plugin_init(void)
 {
 	return (&regex_trans);
 }
diff -up nfs-utils-2.6.2/utils/exportfs/exportfs.c.omv~ nfs-utils-2.6.2/utils/exportfs/exportfs.c
--- nfs-utils-2.6.2/utils/exportfs/exportfs.c.omv~	2022-10-28 16:40:04.991128452 +0200
+++ nfs-utils-2.6.2/utils/exportfs/exportfs.c	2022-10-28 16:40:12.897181253 +0200
@@ -69,14 +69,14 @@ static int _lockfd = -1;
  * need these additional lockfile() routines.
  */
 static void
-grab_lockfile()
+grab_lockfile(void)
 {
 	_lockfd = open(lockfile, O_CREAT|O_RDWR, 0666);
 	if (_lockfd != -1)
 		lockf(_lockfd, F_LOCK, 0);
 }
 static void
-release_lockfile()
+release_lockfile(void)
 {
 	if (_lockfd != -1) {
 		lockf(_lockfd, F_ULOCK, 0);
diff -up nfs-utils-2.6.2/utils/idmapd/idmapd.c.omv~ nfs-utils-2.6.2/utils/idmapd/idmapd.c
--- nfs-utils-2.6.2/utils/idmapd/idmapd.c.omv~	2022-10-28 16:40:29.921294843 +0200
+++ nfs-utils-2.6.2/utils/idmapd/idmapd.c	2022-10-28 16:40:33.263317123 +0200
@@ -867,7 +867,7 @@ nfsdreopen_one(struct idmap_client *ic)
 }
 
 static void
-nfsdreopen()
+nfsdreopen(void)
 {
 	nfsdreopen_one(&nfsd_ic[IC_NAMEID]);
 	nfsdreopen_one(&nfsd_ic[IC_IDNAME]);
diff -up nfs-utils-2.6.2/utils/mount/network.c.omv~ nfs-utils-2.6.2/utils/mount/network.c
--- nfs-utils-2.6.2/utils/mount/network.c.omv~	2022-10-28 16:40:49.642426237 +0200
+++ nfs-utils-2.6.2/utils/mount/network.c	2022-10-28 16:40:59.946494811 +0200
@@ -179,7 +179,7 @@ static const unsigned long probe_mnt3_on
 
 static const unsigned int *nfs_default_proto(void);
 #ifdef MOUNT_CONFIG
-static const unsigned int *nfs_default_proto()
+static const unsigned int *nfs_default_proto(void)
 {
 	extern unsigned long config_default_proto;
 	/*
@@ -192,7 +192,7 @@ static const unsigned int *nfs_default_p
 	return probe_tcp_first; 
 }
 #else
-static const unsigned int *nfs_default_proto() 
+static const unsigned int *nfs_default_proto(void) 
 {
 	return probe_tcp_first; 
 }
