diff -up nfs-utils-2.7.1/support/nfs/exports.c.2~ nfs-utils-2.7.1/support/nfs/exports.c
--- nfs-utils-2.7.1/support/nfs/exports.c.2~	2024-08-22 00:54:50.118548795 +0200
+++ nfs-utils-2.7.1/support/nfs/exports.c	2024-08-22 00:55:22.638706463 +0200
@@ -634,7 +634,7 @@ void fix_pseudoflavor_flags(struct expor
 static int
 parseopts(char *cp, struct exportent *ep, int *had_subtree_opt_ptr)
 {
-	int	had_subtree_opt = 0;
+	int	had_subtree_opt = 1;
 	char 	*flname = efname?efname:"command line";
 	int	flline = efp?efp->x_line:0;
 	unsigned int active = 0;
