diff -up nfs-utils-2.6.4/utils/gssd/gssd_proc.c.omv~ nfs-utils-2.6.4/utils/gssd/gssd_proc.c
--- nfs-utils-2.6.4/utils/gssd/gssd_proc.c.omv~	2023-11-18 16:47:55.505463390 +0100
+++ nfs-utils-2.6.4/utils/gssd/gssd_proc.c	2023-11-18 16:52:42.616371244 +0100
@@ -412,6 +412,7 @@ create_auth_rpc_client(struct clnt_info
 		tid, tgtname);
 	auth = authgss_create_default(rpc_clnt, tgtname, &sec);
 	if (!auth) {
+#if 0 // struct rpc_gss_sec doesn't actually have a minor_status field...
 		if (sec.minor_status == KRB5KRB_AP_ERR_BAD_INTEGRITY) {
 			printerr(2, "WARNING: server=%s failed context "
 				 "creation with KRB5_AP_ERR_BAD_INTEGRITY\n",
@@ -428,6 +429,7 @@ create_auth_rpc_client(struct clnt_info
 					goto success;
 			}
 		}
+#endif
 		/* Our caller should print appropriate message */
 		printerr(2, "WARNING: Failed to create krb5 context for "
 			    "user with uid %d for server %s\n",
