diff -up nfs-utils-2.5.4/configure.ac.3~ nfs-utils-2.5.4/configure.ac
--- nfs-utils-2.5.4/configure.ac.3~	2021-08-09 19:19:11.410981102 +0200
+++ nfs-utils-2.5.4/configure.ac	2021-08-09 19:21:15.974703120 +0200
@@ -638,47 +638,15 @@ AC_SUBST(CPPFLAGS_FOR_BUILD)
 AC_SUBST(LDFLAGS_FOR_BUILD)
 
 my_am_cflags="\
- -pipe \
  -Wall \
  -Wextra \
  $rpcgen_cflags \
- -Werror=missing-prototypes \
- -Werror=missing-declarations \
- -Werror=format=2 \
- -Werror=undef \
- -Werror=missing-include-dirs \
- -Werror=strict-aliasing=2 \
- -Werror=init-self \
- -Werror=implicit-function-declaration \
- -Werror=return-type \
- -Werror=switch \
- -Werror=overflow \
- -Werror=parentheses \
- -Werror=aggregate-return \
- -Werror=unused-result \
  -fno-strict-aliasing \
 "
 
-AC_DEFUN([CHECK_CCSUPPORT], [
-  my_save_cflags="$CFLAGS"
-  CFLAGS="-Werror $1"
-  AC_MSG_CHECKING([whether CC supports $1])
-  AC_COMPILE_IFELSE([AC_LANG_PROGRAM([])],
-    [AC_MSG_RESULT([yes])]
-    [$2+=$1],
-    [AC_MSG_RESULT([no])]
-  )
-  CFLAGS="$my_save_cflags"
-])
-
-CHECK_CCSUPPORT([-Werror=format-overflow=2], [flg1])
-CHECK_CCSUPPORT([-Werror=int-conversion], [flg2])
-CHECK_CCSUPPORT([-Werror=incompatible-pointer-types], [flg3])
-CHECK_CCSUPPORT([-Werror=misleading-indentation], [flg4])
-CHECK_CCSUPPORT([-Wno-cast-function-type], [flg5])
 AX_GCC_FUNC_ATTRIBUTE([format])
 
-AC_SUBST([AM_CFLAGS], ["$my_am_cflags $flg1 $flg2 $flg3 $flg4 $flg5"])
+AC_SUBST([AM_CFLAGS], ["$my_am_cflags"])
 
 # Make sure that $ACLOCAL_FLAGS are used during a rebuild
 AC_SUBST([ACLOCAL_AMFLAGS], ["-I $ac_macro_dir \$(ACLOCAL_FLAGS)"])
